backend/project/utils/submissions/evaluator.py

```diff
@@ -14,8 +14,12 @@
 from project.db_in import db
 from project.models.submission import Submission
 
+
+EVALUATORS_FOLDER = path.join(path.join(__file__), "evaluators")
```

Comment:
Based on what was removed.
```suggestion
EVALUATORS_FOLDER = path.join(path.dirname(__file__), "evaluators")
```

Reply:
Good catch, fixed in [17597fe](https://github.com/SELab-2/UGent-3/pull/256/commits/17597feaa93539f4a5d3e0514548f53c90ae78ac)